### ISSUE TITLE
Is there a sensible way to disable docstring extraction?

### ISSUE THREAD

**Issue by TheMohni** [tfranzel/drf-spectacular#1161]
The classes, methods and properties of my application have docstrings that are meant for internal use only. I dont want any of them to be visible in the API documentation. I know that there is a setting `GET_LIB_DOC_EXCLUDES`, but that only works for classes. I can also subclass `AutoSchema`, but the `get_doc` function that does the extraction is used in methods that do not seem to be designed to be overridden.

Is there an intended way to do this?

**Comment by tfranzel** [comment]
First of all I would like to highlight that the information extraction is apparently working better than anticipated  🤣 This issue has not come up yet before.

It was assumed that `GET_LIB_DOC_EXCLUDES` would be sufficient to suppress unwanted docstrings. `get_doc` doesn't even see all the description strings the system processes (I believe). 

What unwanted strings are left if you properly set `GET_LIB_DOC_EXCLUDES` and overwrite `get_description` via custom `AutoSchema`?

**Comment by tfranzel** [comment]
> I could override the problematic fields in serializers and add empty help_text or use extend_schema

yes that would be too cumbersome.

btw, model fields `help_text` will not go through any of those mechanisms, i.e. patching `get_doc` will do nothing for that case. 

You could write a `POSTPROCESSING_HOOK` (see docs) to clear out unwanted string, but seems a little hacky too.

**Comment by TheMohni** [comment]
> btw, model fields help_text will not go through any of those mechanisms, i.e. patching get_doc will do nothing for that case.

Yeah, I know. Model fields help_text are not an issue for us. We have used them as a help text for users also, so they are in fact almost always useful as API documentation and they don't contain the weird stuff our docstrings have.

I took a quick look at the `POSTPROCESSING_HOOK`, but I don't see an easy way to use it. If I understood correctly, it gets fully generated schema so there is no way to tell which descriptions come from docstrings and which come from somewhere else.

**Comment by tfranzel** [comment]
Although the functionality is also achievable through `GET_LIB_DOC_EXCLUDES`, it requires some non-trivial thinking. I believe this need is not that uncommon, so I agree to add it.
